tikv-client/src/main/java/com/pingcap/tikv/ReadOnlyPDClient.java

```diff
@@ -16,52 +16,19 @@
 
 package com.pingcap.tikv;
 
-import com.google.protobuf.ByteString;
-import com.pingcap.tikv.meta.TiTimestamp;
-import com.pingcap.tikv.region.TiRegion;
 import com.pingcap.tikv.util.BackOffer;
-import java.util.List;
+import com.pingcap.tikv.util.Pair;
 import java.util.concurrent.Future;
+import org.tikv.kvproto.Metapb;
 import org.tikv.kvproto.Metapb.Store;
+import org.tikv.shade.com.google.protobuf.ByteString;
 
 /** Readonly PD client including only reading related interface Supposed for TiDB-like use cases */
-public interface ReadOnlyPDClient {
-  /**
-   * Get Timestamp from Placement Driver
-   *
-   * @return a timestamp object
-   */
-  TiTimestamp getTimestamp(BackOffer backOffer);
+public interface ReadOnlyPDClient extends org.tikv.common.ReadOnlyPDClient {
 
-  /**
-   * Get Region from PD by key specified
-   *
-   * @param key key in bytes for locating a region
-   * @return the region whose startKey and endKey range covers the given key
-   */
-  TiRegion getRegionByKey(BackOffer backOffer, ByteString key);
+  Future<Pair<Metapb.Region, Metapb.Peer>> getRegionByKeyAsync(BackOffer backOffer, ByteString key);
```

Comment:
It's better to keep the comment

Reply:
Is this necessary? When not more people are using this, I prefer to align with upstream.